internal/bundle/helpers.go

```diff
@@ -68,3 +76,169 @@ func destroyBundle(t *testing.T, path string) error {
 	_, _, err := c.Run()
 	return err
 }
+
+var sparkVersions = []string{
+	"11.3.x-scala2.12",
+	"12.2.x-scala2.12",
+	"13.0.x-scala2.12",
+	"13.1.x-scala2.12",
+	"13.2.x-scala2.12",
+	"13.3.x-scala2.12",
+	"14.0.x-scala2.12",
+	"14.1.x-scala2.12",
+}
```

Comment:
this list will get out of date in couple of months, can you use the `w.Clusters.SparkVersions` to pick the list of runtimes dynamically?
see https://github.com/databricks/databricks-sdk-py/blob/main/tests/integration/test_auth.py#L90-L124

Reply:
I consider that but for now we want to keep it simple since we don't run them continuously yet. Once we do, we will get the list dynamically indeed